extensions/cacher/README.md

```diff
@@ -0,0 +1,36 @@
+<p align="center">
+  <img src="media/cacher-logo.png" height="128">
+  <h1 align="center">Cacher Extension for Raycast</h1>
+</p>
+
+## About Cacher
+
+[Cacher](https://cacher.io) is a cross-platform, cloud-based app used to curate a snippet library for you and your team. This extension lets you quickly create and insert personal/team snippets using the [Raycast](https://www.raycast.com) launcher.
+
+## Getting Started
+
+1. [Click here](https://app.cacher.io/enter?action=view_api_creds) to view your Cacher API credentials.
+2. Fill in the **API Key** and **API Token** fields for the Cacher Raycast extension.
+3. Once you're signed into Cacher, you can trigger the **Search Code Snippets** and **Create Code Snippet** commands from Raycast.
+
+![Cacher API Credentials](media/api-credentials.png "Cacher API Credentials")
+
+![Cacher Extension Settings](media/raycast-cacher-settings.png "Cacher Extension Settings")
+
+## Extension Commands
+
+### 🔍 Search Code Snippets
+
+Search Cacher snippets across your personal and team libraries with support for filtering by labels. Easily copy/paste or insert code into your editor.
+
+![Search Code Snippets](media/search-code-snippets.png "Search Code Snippets")
+
+### ✏️ Create Code Snippet
+
+Quickly create snippets for personal or team use.
+
+![Create Code Snippet](media/create-code-snippet.png "Create Code Snippet")
```

Comment:
You can reference this directly from metadata (so you only have them one place, remember to add the optimized to metadata and remove them from media folder)

Reply:
Moved to `metadata/`.